it/router/RouterAuditFeature.scala

```diff
@@ -249,17 +323,21 @@ class RouterAuditFeature extends StubbedFeatureSpec with CommonStubs {
 
   def verifyAuditEvent(auditEventStub: RequestPatternBuilder, expectedReasons: mutableMap[String, String], expectedTransactionName: String, ruleApplied: String): Unit = {
     val loggedRequests = WireMock.findAll(auditEventStub).asScala.toList
-    val event = Json.parse(loggedRequests.filter(s => {println(s.getBodyAsString);s.getBodyAsString.matches( """^.*"auditType"[\s]*\:[\s]*"Routing".*$""")}).head.getBodyAsString)
+    val event = Json.parse(loggedRequests.filter(s => {
+      println(s.getBodyAsString);
```

Comment:
this println has been probably left in one of the previous commits, it's not needed